File: ktx-api/src/main/kotlin/com/omerilhanli/api_ktx/model/common/State.java
package com.omerilhanli.api_ktx.model.common;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class State implements Serializable {

    @SerializedName("day")
    @Expose
    private int day;

    @SerializedName("time")
    @Expose
    private String time;

    public int getDay() {
        return day;
    }

    public void setDay(int day) {
        this.day = day;
    }

    public String getTime() {
        return time;
    }

    public void setTime(String time) {
        this.time = time;
    }
}

File: ktx-api/src/main/kotlin/com/omerilhanli/api_ktx/model/common/State.kt
package com.omerilhanli.api_ktx.model.common

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class State(
    @SerializedName("day")
    @Expose
    var day: Int? = 0,
    @SerializedName("time")
    @Expose
    var time: String? = null
): Serializable